### DISCUSSION TITLE
Dependency calls for azure service bus

### DISCUSSION THREAD

**Discussion by ShriramDeshkar** [open-telemetry/community#1673] [Community Discussions]
I m using Java azure function written in spring boot framework. I have enabled auto instrumentation through configuration changes for open telemetry with azure monitor . I can able to see data to azure monitor. Also able to see dependencies chart for services which I m calling from azure function like storage account, redis cache but I m not able to see dependency for azure service bus . Is it supported?

**Comment by trask** [comment]
hi @ShriramDeshkar! I'd recommend asking in https://github.com/open-telemetry/opentelemetry-java-instrumentation
